app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt

```diff
@@ -14,6 +14,18 @@ private const val KEY_SKILL_ID_LIST = "SKILL_ID_LIST"
 
 /** Fragment that displays skills for topic practice mode. */
 class TopicPracticeFragment : InjectableFragment() {
+  companion object {
+    /** Returns a new [TopicPracticeFragment] */
```

Comment:
Add full-stop at the end of the sentence `/** Returns a new [TopicPracticeFragment]. */`

Reply:
Done